src/tools/gophish_complete.py

```diff
@@ -46,46 +48,66 @@
 requests.packages.urllib3.disable_warnings()
 
 
-def get_campaigns(api):
-    """Return a dictionary containing all campaigns."""
+def get_campaign_id(campaign_name, campaigns):
+    """Get campaign id from campaign name.
+
+    Args:
+        campaign_name (string): Full campaign name.
+        campaigns (dict): Campaign id as key, campaign name as value.
+
+    Raises:
+        LookupError: When the campaign name is not found, raise exception.
+
+    Return:
+        Return campign id that corresponds to campaign name provided.
```

Comment:
Match verb tense with rest of docstring and eliminate redundant words:
```
Returns:
    Campaign id corresponding to campaign name provided.
```

Reply:
Fixed in commit 1ab0c81.